rocky/rocky/views/mixins.py

```diff
@@ -46,6 +48,19 @@ class OriginData(BaseModel):
     boefje: Boefje | None = None
     params: dict[str, str] | None = None
 
+    @property
+    def is_old(self, maxage: Optional[int] = None) -> bool
+        if not maxage:
+            maxage = ORIGIN_MAX_AGE
+        if not self.normalizer:
```

Comment:
```suggestion
        if self.normalizer is None:
```
or leave as is, if you're specifically validating against both a `None` and an empty dict

Reply:
Since the job might not have a normalizer (due to various errors), I was trying to do some defensive coding.